.github/workflows/sycl-linux-precommit.yml

```diff
@@ -128,6 +128,37 @@ jobs:
       ref: ${{ github.sha }}
       merge_ref: ''
 
+      sycl_toolchain_artifact: sycl_linux_default
+      sycl_toolchain_archive: ${{ needs.build.outputs.artifact_archive_name }}
+      sycl_toolchain_decompress_command: ${{ needs.build.outputs.artifact_decompress_command }}
+  test-esimd:
```

Comment:
I'd prefer this merged into the matrix job at line 50. Not sure how to do it best.